echo-simulcast/index.html

```diff
@@ -0,0 +1,362 @@
+    <script
+      src="https://cdnjs.cloudflare.com/ajax/libs/webrtc-adapter/8.1.2/adapter.min.js"
+      integrity="sha512-l40eBFtXx+ve5RryIELC3y6/OM6Nu89mLGQd7fg1C93tN6XrkC3supb+/YiD/Y+B8P37kdJjtG1MT1kOO2VzxA=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    ></script>
+  </head>
+
+  <body>
+    <div class="grid">
+      <h1>Calls Echo Demo</h1>
+      <div>
+        <h2>Local stream</h2>
+        <video id="local" autoplay muted playsinline></video>
+      </div>
+      <div>
+        <h2>Remote echo stream</h2>
+        <video id="remote" autoplay muted playsinline></video>
+      </div>
+    </div>
+
+    <script type="module">
+      // This is the App ID from the dashboard that identifies this Calls Application.
+      // https://dash.cloudflare.com/?to=/:account/calls
+      // Note: Calls sessions are limited to interacting with sessions in the same App ID.
+      const APP_ID = "$APP_ID";
+      // ❗❗❗ DO NOT USE YOUR TOKEN IN THE BROWSER FOR PRODUCTION. It should be kept and used server-side.
+      const APP_TOKEN = "$APP_TOKEN";
+      // We'll use this for authentication when making requests to the Calls API.
+      const headers = {
+        Authorization: `Bearer ${APP_TOKEN}`,
+      };
+      const API_BASE = `https://rtc.live.cloudflare.com/v1/apps/${APP_ID}`;
+      const localVideo = document.querySelector("video#local");
+      const remoteVideo = document.querySelector("video#remote");
+      if (
+        !(localVideo instanceof HTMLVideoElement) ||
+        !(remoteVideo instanceof HTMLVideoElement)
+      )
+        throw new Error("These should be video elements");
+
+      // get a MediaStream from the user's webcam & mic
+      const media = await navigator.mediaDevices.getUserMedia({
+        audio: true,
+        video: { height: {ideal: 720}}
+      });
+      // show the local feed
+      localVideo.srcObject = media;
+
+      // First, we'll establish the "local" Calls session by calling createCallsSession
+      // which is defined towards the bottom of this script. This will create an
+      // a Calls session, and return the session ID.
+      const localSessionId = await createCallsSession();
+
+      // Then we create a simple RTCPeerConnection with some standard parameters.
+      const localPeerConnection = await createPeerConnection();
+
+      // Next we need to push our audio and video tracks. We will add them to the peer
+      // connection using the addTransceiver API which allows us to specify the direction
+      const transceivers = media.getTracks().map((track) => {
+        const trackInit = { direction: 'sendonly' }
+        // If the track is a video track we are going to enable three simulcast layers.
+        // You can name the RIDs how ever you like, but the automatic modes of the SFU
+        // expects to be able to sort them alphabetically.
```

Comment:
Should this say `asciibetically`? 😅

Reply:
Also might be good to specify which direction in the asciibetical sorting corresponds to higher/lower quality?